### ISSUE TITLE
Font Awesome 4.7.0 --> Font Awesome 6.x upgrade possible

### ISSUE THREAD

**Issue by kees-closed** [getgrav/grav-theme-quark#200]
Hi, is it possible to upgrade the Font Awesome of this theme to the latest version? I suppose this is delayed due to incompatibilities? But by now I think the missing icons or out of date icons are also not ideal. I hope you guess have enough time to consider and implement this. Thanks.

**Comment by kees-closed** [comment]
It's been a while and the current icon set is getting more dated with each year. I built my websites on Quark, since it was the default theme and also used by the Admin panel. But it doesn't seem well maintained? Most development goes into the paid themes? I'm willing to pay for active maintenance on some things. But I already bought several plugins in support and donated.

Isn't this icon set upgrade as simple as just replacing the icon set? Of course things can break due to changed icon labels or removed icons. But that too is a simple replace of that label, right?

**Comment by kees-closed** [comment]
This is the official theme of Grav, right? It's the one that the Admin Panel also uses. Would it be a lot of work to update the icons?

Based on an [open PR](https://github.com/getgrav/grav-theme-quark/pull/152/files), it's just a matter of updating a bunch of files, which are likely just downloadable somewhere? Or is the issue more that icon labels are not backwards compatible when upgrading it to the latest version?

If any help is needed, let me know.

**Comment by GastonGH** [comment]
Yes, an upgrade to the current version would be nice, as there are already some new or changed icons—and I'm not just thinking of Facebook and the like. 
Thanks!
